packages/atomic-hosted-page/src/components/atomic-hosted-page/hosted-pages.ts

```diff
@@ -0,0 +1,67 @@
+export interface HostedPage {
```

Comment:
All the ref doc in this file is copied from the [Platform Client PR](https://github.com/coveo/platform-client/pull/616) and reviewed